front/src/components/InsertPositionsPanel/PositionRow/PositionRow.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+
+const PositionRow = (props) => {
+    let [position, setPosition] = useState(props.position);
+    let [order, setOrder] = useState(props.order);
+
+    const handlePositionChange = (event) => {
+        setPosition(event.target.value);
+    };
```

Comment:
If position value is supposed to be in range [min, max] (I believe it is, since there are min and max fields in props), consider adding value checker. Currently position value can exceed desired scope, as one could simply type number in, instead of choosing it with arrow-buttons.
```suggestion
   const handlePositionChange = (event) => {
        let pos = event.target.value;
        if (pos > props.max) pos = props.max;
        else if (pos < props.min) pos = props.min;
        setPosition(pos);
    };

```

Reply:
I believe clamp could be simplified to oneliner:
```suggestion
   const handlePositionChange = (event) => {
        let pos = Math.min(Math.max(event.target.value, props.min), props.max);
        setPosition(pos);
    };

```

Reply:
You're right, already fixed an commited that